CMakeLists.txt

```
@@ -133,8 +133,11 @@ if(MSVC)
       string(REPLACE "/RTC1" "" CMAKE_CXX_FLAGS_DEBUG  "${CMAKE_CXX_FLAGS_DEBUG}")
       string(REPLACE "/RTC1" "" CMAKE_C_FLAGS_DEBUG  "${CMAKE_C_FLAGS_DEBUG}")
     else()
-      # Squeeze binary in Release builds
-      add_link_options(/OPT:REF /OPT:ICF)
+      # Reduce the size of the binary in Release & RelWithDebInfo builds
+      add_link_options(/OPT:REF)
```

```suggestion
      add_link_options(/OPT:REF  /OPT:ICF)
```
We need somewhere document our decisions for the future. This PR will be out of sight when someone stumbles over it. 
So we may either just enable /OPT:ICF
or